docs/user_guide.md

```diff
@@ -668,6 +668,8 @@ optional arguments:
                         reports to the database. If not specified, the '--
                         name' parameter given to 'codechecker-analyze' will be
                         used, if exists.
+  --tag TAG             An identifier for this individual store of results in
```

Comment:
A more descriptive help message would be better to note to the user that an already used tag can not be used again.